tests/test_hugo_server_readiness.py

```
@@ -0,0 +1,161 @@
+    with patch.object(manager, "_check_process_alive", return_value=True), patch.object(
+        manager, "_check_port_listening", return_value=True
+    ):
+        manager.is_running = True
+        manager.pid = 12345
+        status = manager.get_status()
+        assert status["running"] is True
+        assert status["ready"] is True
+
+
+def test_get_status_ready_false_when_port_not_open(manager):
+    """进程存活但端口未监听时 get_status 返回 ready=False（正在构建）。"""
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)